fundamentals/problem-set-conditionals.md

```diff
@@ -516,43 +516,43 @@ else:
 <!-- prettier-ignore-start -->
 ### !challenge
 
-* type: checkbox
-* id: a61f06a0-94aa-4617-89a2-86f1217c8ef4
-* title: Boolean practice
+* type: multiple-choice
+* id: 4c8d3f3b-3edd-4269-ad7a-90e9bac49685
+* title: Boolean Practice
 
 ##### !question
 
 Voice assistants like Alexa will turn on when any of its "wake words" is announced nearby. Which option best describes this behavior?
```

Comment:
```suggestion
Voice assistants like Alexa will turn on when any of its "wake words" are announced nearby. Which option best describes this behavior?
```

This was already existing, but would be nice to match plural case of "wake words" here